bin/fix/fix-audit.rb

```diff
@@ -0,0 +1,188 @@
+require 'rest-client'
+# we use active_fedora/noid to generate pairtrees from noids
+require 'active_fedora/noid'
+require 'yaml'
+require 'rdf/turtle'
+require File.expand_path('../../../config/environment', __FILE__)
+
+def read_config
+  rails_env = Rails.env
+  config = YAML.load_file("config/fedora.yml")
+  @user = config[rails_env]['user']
+  @password = config[rails_env]['password']
+  @rest = config[rails_env]['url']
+  @base_path = config[rails_env]['base_path']
+end
+
+def patch(path, payload)
+  puts 'Patching ' + path + ': ' + payload
```

Comment:
@mbarnett I guess I didn't think about it too deeply. Lots of puts usually means that using a logger is the next step. Realistically your solution would have the same result - with more chance for the operator to forget to pipe to file. I'm assuming that we will want to save this output for its potential to help us debug future prolems. But maybe we don't need to be worried about this here.